packages/theme/.history/.env_20220228182317

```diff
@@ -0,0 +1,2 @@
+GRAPHQL_API=http://localhost:3000/shop-api
```

Comment:
Could you please remove these .history files? I dont know where they came from but they are not needed for sure

Reply:
Sure I'll remove them.